src/Microsoft.OData.Core/Value/ODataStreamReferenceValue.cs

```diff
@@ -61,7 +62,8 @@ public Uri ReadLink
             {
                 return this.HasNonComputedReadLink
                     ? this.readLink
-                    : (this.computedReadLink ?? (this.metadataBuilder == null ? null : this.computedReadLink = this.metadataBuilder.GetStreamReadLink(this.edmPropertyName)));

```

Comment:
>this. [](start = 150, length = 5)

I cannot figure the reason to change this.

Reply:
Fixed.

---
In reply to: [266668059](https://github.com/OData/odata.net/pull/1204#discussion_r266668059) [](ancestors = 266668059)